src/components/BoardHeader/ParticipantsList/ProgressCircle.tsx

```diff
@@ -0,0 +1,25 @@
+import {VFC} from "react";
+
+interface ProgressCircleProps {
```

Comment:
I think we should agree in the near future to use either `interface` or `type` (for consistency).